Improve branding

owtaylor opened the pull request:
* show Granite.Code for in the status bar, not Continue
* Use the Granite.Code icons for the icon in the sidebar and for the extension icon.

![sidebar-screenshot](https://github.com/user-attachments/assets/965d52ec-5be7-44e5-a41e-640c674b846c)

I used the outlined version of the icon for the sidebar, since it seemed more consistent with the general vscode icon style, though it's a bit thin.

owtaylor committed:
Granite.Code: adjust extension name in status bar

owtaylor committed:
Granite.Code: Add Granite.Code icon and sidebar icon

fbricon commented on extensions/vscode/src/autocomplete/statusBar.ts:
```diff
@@ -47,20 +47,20 @@ const statusBarItemText = (
 ) => {
   if (error) {
-    return "$(alert) Continue (FATAL ERROR)";
+    return "$(alert) Granite.Code (FATAL ERROR)";
```
maybe use a constant everywhere for the name. 
Might make sense to add an EXTENSION_LABEL constant in core/control-plane/env.ts and upstream the change

owtaylor replied:
I don't think it makes sense *unless* we can upstream it, since otherwise we create double edits (one to import, one to use). If we can upstream it, yeah, it would be better.

fbricon commented:
Also changing the context menus as part of this PR might make sense.
<img width="570" alt="Screenshot 2025-02-12 at 11 19 22" src="https://github.com/user-attachments/assets/077a0307-19d2-42b1-809a-c80ab8f1949f" />

owtaylor commented:
> Also changing the context menus as part of this PR might make sense.

This one gets taken care of when we transform package.json to create the Granite.Code extension:

<img width="588" alt="image" src="https://github.com/user-attachments/assets/cdddaed3-f489-4129-a905-ffceee0f0f47" />

owtaylor committed:
Granite.Code: Change label for default quick action

When experimental quick actions are enabled (Continue: Enable Quick Actions), the
default action is labeled "Continue" - switch to "Granite.Code". (These labels
are not great from a UX point of view - it's not clear that the meaning is
*edit with Granite.Code*.)
